spec/support/factory/accounts/ueberauth_factory.ex

```diff
@@ -0,0 +1,106 @@
+# Ueberauth response mock for vk provider
+# insert/1 cannot be used with this factory as it's not in a database
+#
+defmodule Squadster.Support.Factory.Accounts.UeberauthFactory do
+  defmacro __using__(_opts) do
+    quote do
+      import Squadster.Helpers.Dates, only: [date_to_string: 1]
+
+      def ueberauth_factory do
+        # should be integer, cannot start with zero
+        uid = Enum.random(100_000_000..999_999_999)
```

Comment:
Why you reject my approach, I still think that it was better than typing it manually. Although, nothing bad here, but maybe make some kind of helper, that generates this number, instead of calling it multiply times.

Reply:
Oh, I see that it must be int, but pay attention to second part of comment please

Reply:
We'll have to create a helper module for one-line function, I think it's better to leave it as is, at least for now

Reply:
It's ok. Lets keep it as it is